service/db/import.py

```diff
@@ -374,8 +398,13 @@ async def main():
         format="%(asctime)s:%(name)s:%(levelname)s:%(message)s",
     )
 
-    for directory in args.dirs:
-        await import_directory(directory)
+    for path in args.paths:
+        if path.is_dir():
+            await import_directory(path)
+        elif path.suffix == ".zip":
```

Comment:
To be on the safe side for Windows crowd, can you do `path.suffix.lower()` here?